Makefile

```diff
@@ -0,0 +1,15 @@
+all: test
```

Comment:
Hey, I removed the Makefile from this PR as to not keep us back from merging this. I separated it to a new PR in https://github.com/ipfs/go-ipfs-dep/pull/15. 

This was a good discussion and at least the link to it should be kept safe for future references :)